src/common/Surface.js

```diff
@@ -59,6 +59,11 @@
     Surface.prototype.publish("buttonAnnotations", [], "array", "Button Array",null,{tags:["Intermediate"]});
     Surface.prototype.publish("buttonGutter", 25, "number", "Space Between Menu and Buttons",null,{tags:["Intermediate"]});
 
+    Surface.prototype.publish("showContent", true, "boolean", "Show Content",null,{tags:["Intermediate"]});
+    Surface.prototype.publish("menu", [], "array", "Menu List Data",null,{tags:["Intermediate"]});
+    Surface.prototype.publish("menuPadding", 2, "number", "Menu Padding",null,{tags:["Advance"]});
```

Comment:
Once properties are published the "overriding" methods need to call to "new" corresponding base class methods.